configure.ac

```diff
@@ -380,9 +379,10 @@ AS_IF([test "x$PACKAGE_STATUS" = xdevelopment],
 AS_IF([test "x$PLATFORM" = xosx],
     [AM_CFLAGS="$AM_CFLAGS -Qunused-arguments"])
 
-AM_CFLAGS="$AM_CFLAGS $PTHREAD_CFLAGS $glib_CFLAGS $gio_CFLAGS $curl_CFLAGS ${SQLITE_CFLAGS} $libstrophe_CFLAGS"
```

Comment:
do we need the libstrophe cflags? when do they get passed?

Reply:
they get put into `CFLAGS` here https://github.com/profanity-im/profanity/blob/f3265565e853ecf993f12093c6a0c94acf3f212e/configure.ac#L155-L156